rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import eslint from '@rollup/plugin-eslint'
+import json from '@rollup/plugin-json'
+import replace from '@rollup/plugin-replace'
+import url from '@rollup/plugin-url'
+import svgr from '@svgr/rollup'
+import sass from 'rollup-plugin-scss'
+import typescript from 'rollup-plugin-typescript2'
+
+import { dependencies } from './package.json'
+
+const deps = Object.keys(dependencies)
+
+const replacements = {
+  'process.env.REACT_APP_IS_WIDGET': true,
+}
+
+// This is necessary because some nested imports (eg jotai/*) would otherwise not resolve.
+function external(source) {
+  const dep = deps.find((dep) => source === dep || source.startsWith(dep + '/'))
+  return Boolean(dep)
+}
+
+const config = {
+  input: 'src/lib/index.tsx',
+  output: [
+    {
+      file: 'dist/interface.js',
```

Comment:
iiuc, this is the name of the output lib? should it be called something other than `interface`? `bundle`, `main`, `widget`, etc

Reply:
I guess you're matching the metadata in `package.json`.. i think i would rename all but not a blocker